### PR TITLE
[Documentation][Resource] Fix typos in docs

### PR DESCRIPTION
**Pull request by harikt**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | none |
| License         | MIT |

### CODE REVIEW

File: docs/components/Resource/translation.rst [hunk 1]
```diff
@@ -135,21 +135,21 @@ When we already have our translations, we can work with the **Book**:
...
 
-You can always use the ``getTranslation`` method by itself, but the same principal is in play:
+You can always use the ``translate`` method by itself, but the same principal is in play:
 
 .. code-block:: php
 
    <?php
 
-   $harryPotter->getTranslation('de');  // returns $germanBook
+   $harryPotter->translate('de');  // returns $germanBook
```
Comment (lchrusciel) [inline docs/components/Resource/translation.rst]: This as well

File: docs/components/Resource/translation.rst [hunk 2]
```diff
@@ -65,15 +65,15 @@ Now the following class will be actually capable of translating the **title**:
         */
        public function getTitle()
        {
-           return $this->getTranslation()->getTitle();
+           return $this->getTranslations()->getTitle();
```
Comment (lchrusciel) [inline docs/components/Resource/translation.rst]: Shouldn't be changed

**Comment by harikt** [comment]
Hm.. Looks like master changed a bit. https://github.com/Sylius/Sylius/blob/6774a9396f4a2ac68200e965776322c60952bce4/src/Sylius/Component/Resource/Model/TranslatableTrait.php#L61 . I was using composer and it installed 

```
"name": "sylius/resource",
"version": "v0.19.0",
```

I will look into and make the change.

**Comment by harikt** [comment]
Opened https://github.com/Sylius/Sylius/pull/8205 with the change.

**Pull request closed without merging**